packages/client/src/lib/interface.ts

```diff
@@ -46,6 +46,10 @@ export interface API {
    * Stores a single file and returns a corresponding CID.
    */
   storeBlob(service: Service, content: Blob | File): Promise<CIDString>
+  /**
+   * Stores CAR file and returns a corresponding CID.
+   */
+  storeCar(service: Service, content: Blob | File): Promise<CIDString>
```

Comment:
I think this takes an options object too now.

Reply:
like dis?

```ts
  storeCar(
    service: Service,
    content: Blob | File,
    options?: { onStoredChunk?: (size: number) => void }
  ): Promise<CIDString>
```